Makefile

```diff
@@ -256,7 +256,7 @@ endif
 ifeq ($(BASE_VERTICA_IMG), <not-set>)
```

Comment:
Can we change the name of the Makefile target too?

Reply:
Must have missed this one, will change